ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/AdminResponseHandlers.java

```diff
@@ -326,4 +342,26 @@ static boolean isConnectErrorResponse(final JsonObject ksqlEntity) {
       return Optional.empty();
     }
   }
+
+  /**
+   * Attempts to parse the provided response entity as a {@code ServerInfo}.
+   *
+   * @param serverInfoEntity response entity
+   * @return optional containing parsed result if successful, else empty
+   */
+  private static Optional<ServerInfo> getServerInfoResponse(
```

Comment:
We can inline this method. The reason the others aren't inlined is because they may be returned from the generic `/ksql` endpoint, in which case it's useful to have a separate helper method to check the type.

Or, if you think it's cleaner to have a separate method rather than inlining this, we should at least update the return type to not be optional. There is no situation in which we expect this method to be called and not result in a `ServerInfo` object.